CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.45.1 / 2021-02-01
+
+* [BUGFIX] Generate receivers from alertmanagerconfig even if routes are empty. #3819
```

Comment:
nit: this is not necessary as we generate the changelog from the PR description.

Reply:
Thank you @s-urbaniak , will remove this